src/components/utils/custom-nested-menu.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useState } from 'react';
+import { NestedMenuItem } from 'mui-nested-menu';
+import { Box } from '@mui/system';
+import MenuItem from '@mui/material/MenuItem';
+import { mergeSx } from './functions';
+
+const styles = {
+    highlightedParentLine: (theme) => ({
+        backgroundColor: theme.palette.action.hover,
+        color: theme.palette.primary.main,
+        transition: 'all 300ms ease',
+    }),
+    highlightedLine: (theme) => ({
+        transition: 'all 300ms ease',
+        '&:hover': {
+            backgroundColor: theme.palette.action.hover,
+            color: theme.palette.primary.main,
+            transition: 'all 300ms ease',
```

Comment:
you can remove this line, it is not necessary to add this css property two times (line 20) to the same class

Reply:
Yes, done.